functions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+function dbpull():array{
```

Comment:
i would say rename the function to `getAllCountriesFromDb` to make it clear exactly what it retrieves

Reply:
or CountryCards or whatever you prefer to call them